src/app/busca-ativa/citopatologico/Cito.tsx

```
@@ -37,63 +36,54 @@ export const Cito: React.FC<CitoProps> = ({
-				? "aps"
-				: "equipe";
-				const newUrl = `${path}?aba=${""}&sub_aba=${activeTabIndex}&visao=${visao}`;
-				if (window.location.search !== new URL(newUrl, window.location.origin).search) {
-					router.replace(newUrl); // Usa replace ao invés de push para evitar loops
-				}
-	}, [activeTabIndex, path, session, router.replace]);
-
-	if (!session) return <Spinner />;
+		router.push(`${path}?aba=${""}&sub_aba=${activeTabIndex}&visao=${visao}`);
+	}, [activeTabIndex, activeTitleTabIndex]);
+	useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: voltarGatilho

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)